src/API/session.js

```diff
@@ -22,7 +22,7 @@ export default {
     } else {
       return fetchJson('/api/development_login', {
         data: { email },
-        method: 'POST',
+        method: 'POST'
```

Comment:
I'm not sure about these automatic formatting changes. It doesn't conform to airbnb style. https://github.com/airbnb/javascript/#commas